recipes/cyavro/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.1" %}
+
+package:
+  name: cyavro
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/MaxPoint/cyavro.git
+  git_tag: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+  script: python setup.py install
+
+requirements:
+    build:
+        - python
+        - cython
+        - numpy
+        - libavro
+        - six
+        - jinja2
+    run:
+        - python
+        - numpy
```

Comment:
Is this linking to `numpy`? Looks like it might be. If so, please add `x.x` after `numpy`.